websites/R/Reddit/presence.ts

```diff
@@ -83,13 +83,23 @@ presence.on("UpdateData", async () => {
       presenceData.smallImageKey = "live";
       presenceData.smallImageText = (await strings).live;
     } else {
-      subReddit = document.querySelector(
+      delete presenceData.smallImageKey
+      let sub = document.querySelector(
         "span._1GieMuLljOrqnVpRAwz7VP"
-      ).textContent;
+      );
+      if (sub == null || !sub) {
+       sub = document.querySelector("#SHORTCUT_FOCUSABLE_DIV > div:nth-child(4) > div > div > div > div._3ozFtOe6WpJEMUtxDOIvtU > div.q4a8asWOWdfdniAbgNhMh > div > div.QscnL9OySMkHhGudEvEya > div > div._3TG57N4WQtubLLo8SbAXVF > h2");
+       if (!sub) {
```

Comment:
wait - what's the point of these ifs

Reply:
I reassigned sub so there are two different elements thats why multiple ifs looks weird tho

Reply:
makes sense but why if sub == null || !sub bruh

Reply:
cause idk